api/v1beta1/foundationdb_version.go

```diff
@@ -153,7 +153,7 @@ func (version FdbVersion) HasSidecarCrashOnEmpty() bool {
 // This is currently set to false across the board, pending investigation into
 // potential bugs with non-blocking excludes.
 func (version FdbVersion) HasNonBlockingExcludes() bool {
-	return version.IsAtLeast(FdbVersion{Major: 6, Minor: 3, Patch: 5})
+	return false
```

Comment:
Like discussed offline I think we should a setting to https://github.com/FoundationDB/fdb-kubernetes-operator/blob/master/docs/cluster_spec.md#automaticreplacementoptions to allow the users to decide what they want to use + allow us to test the different exclusion modes easier. The logic should be in the end:

```go
return version.IsAtLeast(FdbVersion{Major: 6, Minor: 3, Patch: 5}) && useNonBlockingExclude
```